### PR TITLE
Migrate Hippo Assets to the Rackspace CDN

### PR DESCRIPTION
**Pull request by tomas-stefano**
@misaka and @dougdroper This is the first version of the importer. I put in the binary folder and we can run with:

```
ruby bin/migrate_files.rb -u $RACKSPACE_USERNAME -k $RACKSPACE_API_KEY -b $RACKSPACE_BUCKET_NAME -f some_hippo_file.xml
```

### CODE REVIEW

File: bin/migrate_files.rb [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+  def initialize(hippo_files, options)
+    @hippo_files = hippo_files
+    @options = options
+  end
+
+  def upload
+    hippo_files.each do |hippo_file|
+      puts "Uploading #{hippo_file.filename}"
+      hippo_file.blob
```
Comment (tomas-stefano) [inline bin/migrate_files.rb]: I found a bug on the blob method. I will fix but you can review.

File: Gemfile.lock [hunk 2]
```diff
@@ -215,6 +217,89 @@ GEM
...
+    fog-vmfusion (0.0.1)
+      fission
+      fog-core
+    fog-voxel (0.0.2)
+      fog-core
+      fog-xml
+    fog-xml (0.1.1)
+      fog-core
+      nokogiri (~> 1.5, >= 1.5.11)
+    formatador (0.2.5)
```
Comment (tomas-stefano) [inline Gemfile.lock]: Fog uses a lot of dependencies :).

**Comment by tomas-stefano** [comment]
@misaka and @dougdroper can you review this, please? Now the PR it's ready.

**Comment by dougdroper** [comment]
Yes looks good :+1:

**Comment by Tr4pSt3R** [comment]
:+1: looks good to me

**Pull request merged**